contracts/cw1-whitelist/src/contract.rs

```diff
@@ -19,7 +17,10 @@ use crate::state::{AdminList, ADMIN_LIST};
 const CONTRACT_NAME: &str = "crates.io:cw1-whitelist";
 const CONTRACT_VERSION: &str = env!("CARGO_PKG_VERSION");
 
-#[cfg_attr(not(feature = "library"), entry_point)]
+#[cfg(feature="boot")]
+use boot_contract_derive::boot_contract;
+
+#[cfg_attr(feature="boot", boot_contract)]
```

Comment:
If we want to merge this we'll need to re-add the entry_point macro.
Goal of this lib is to be an up-stream cw-plus repo that we could propose to merge when we have a stable release.

Reply:
This is fixed now, thanks